src/components/AddTodoForm.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+const AddTodoForm = () => {
+  return (
+    <form name="AddTodoForm">
+      <center>
+        <h1>Welcome to our Todo Form</h1>
+      </center>
```

Comment:
Please replace or remove the center tag.

I like your thinking about using different types of tags for different content. However, the `center` tag is [deprecated so its usage should be avoided](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/center). The reason why this is, is because `center` was made to be a positional element.

HTML and CSS has came a great way since it was first considered. In current web dev practices, a page is structure is determined by HTML but how it appears is done with CSS. A full list of deprecated HTML elements can be found [at this link on W3 Docs](https://www.w3docs.com/learn-html/deprecated-html-tags.html).